src/components/App/Currencies/CurrenciesList/CurrenciesListContainer.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+
+import CurrenciesList from './CurrenciesList';
+
+const mapStateToProps = state => ({
+  currencies: Object.keys(state.currencies)
+    .map(key => state.currencies[key])
+    .sort(
+      (currencyA, currencyB) => (currencyA.quote.USD.market_cap > currencyB.quote.USD.market_cap ? -1 : 1)
```

Comment:
How do you think maybe it's better to move sorting logic into state module (selector for example)?